### DISCUSSION TITLE
using katana to get urls from specific path

### DISCUSSION THREAD

**Discussion by bahlawi89** [projectdiscovery/katana#233] [Q&A] [answered]
Is it possible to get all URLs under specific path in a domain using katana ? 

Example : 
Test path : https://www.tesla.com/en_CA/

`katana -headless -no-sandbox -show-browser -nc -silent -jc -timeout 25 -retry 2 -fs dn -u https://www.tesla.com/ -cs en_CA`

Expected result : 

```
https://www.tesla.com/en_CA/
https://www.tesla.com/en_ca/support/browser-support
https://www.tesla.com/en_ca/drive
```

**Comment by MetzinAround** [comment]
HI @ehsandeep. Got a question here about katana and getting all the urls in a domain.

**Comment by ehsandeep** [comment, marked as answer]
@bahlawi89, you need to use `-mr` instead of `-cs` to filter the results.

```console
katana -headless -nc -silent -jc -timeout 25 -retry 2 -fs dn -u https://www.tesla.com/en_CA/ -mr en_CA
```
